scripts/deploy.sh

```diff
@@ -46,5 +43,12 @@ key_name=sample-key
 az keyvault key create -n $key_name -p software --vault-name $kv_name --size 2048
 key_version=$(az keyvault key list-versions --name sample-key --vault-name $kv_name | grep "kid" | cut -d '/' -f 6 | cut -d '"' -f 1)
 
-echo -e "\nPlace the following in your config.json:"
-echo '{"azure_tenant_id": "'$tenant_id'", "azure_service_principal_client_id":"'$client_id'", "azure_service_principal_secret":"'$client_secret'", "aes_key_length":32, "aes_iv_length":16, "azure_keyvault_url": "'$keyvault_url'", "azure_keyvault_key_name": "'$key_name'", "azure_keyvault_key_version": "'$key_version'"}' | python -m json.tool
\ No newline at end of file
+echo "export AZURE_TENANT_ID=$tenant_id"
```

Comment:
I think it will be annoying for the user to have to run this deploy script every time they want to play with the samples or have to go manually retrieve and set the variables.

Reply:
I will be surprised if anybody will ever run this demo more than once :)
This is an example on how to use Encryptor, not how to properly store secrets imo

Reply:
I'm not making a case against storing secrets, I'm making a case to keep the config so people can run the demo without having to redeploy the infra.